data/fever/index.html

```diff
@@ -0,0 +1,13 @@
+﻿<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-GB" lang="en-GB">
```

Comment:
html5 doctype vs xhtml

Reply:
Any problem? This is correct as far as I can see.

Reply:
Not really. This is exactly how you're supposed to do XHTML5. Is there a particular reason for using it though?

Reply:
I was just reading through your changes (thanks for then btw!) and stumbled upon this. Was just curious why you would use an xhtml declaration nowadays.

Reply:
Please note that HTML vs. XHTML is not the same question than to use HTML5 vs. HTML4 or XHTML1.

XHTML can be parsed as standard XML easily in almost any language/platform, which is not the case for HTML, even less if it is supposed to be in a consistent manner. This makes XHTML easier to consume. The fact that it can optionally be served strictly makes it easier to spot errors. In deployment, one can serve it as HTML if one does not desire its strictness.

Reply:
* https://html.spec.whatwg.org/multipage/xhtml.html
* https://www.w3.org/TR/html5/xhtml.html
* https://www.w3.org/TR/html-polyglot/#doctype

Reply:
Besides the X/HTML topic, couldn't we use a simple one liner index.php instead?
<?php header('Location: /'):

Reply:
This file is not even supposed to be used. It is mainly there to avoid directory indexing. We have the same template in several other directories. A one-line PHP 301 or 302 redirect is not valid, as per HTTP/1.1 which requires sending a proper body.